From cff40dddea01d217fc3789c2c8e3337ea2ec34b7 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 21 Feb 2011 10:29:47 +0100
Subject: [PATCH] eve: remove unconditional ewk_view_context_menu_forward_event call

* ewk_view_context_menu_forward_event is not available in ewebkit lib (only in API)
  after https://bugs.webkit.org/show_bug.cgi?id=51530
---
 src/bin/view.c |    9 +++++++--
 1 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/src/bin/view.c b/src/bin/view.c
index d743d75..5493b20 100644
--- a/src/bin/view.c
+++ b/src/bin/view.c
@@ -1200,8 +1200,13 @@ _view_smart_mouse_down(Ewk_View_Smart_Data *esd, const Evas_Event_Mouse_Down *ev
    return EINA_TRUE;
 
 forward_event:
-   if (ev->button == 3) // forward of context menu event is special
-      return ewk_view_context_menu_forward_event(sd->base.self, ev);
+   /* https://bugs.webkit.org/show_bug.cgi?id=51530 
+    * made ewk_view_context_menu_forward_event conditionally available
+    * but didn't change it in ewk_view.h, so this compiles fine, but later
+    * linker fails. Remove this part here temporary before proper solution is found
+    * if (ev->button == 3) // forward of context menu event is special
+    *   return ewk_view_context_menu_forward_event(sd->base.self, ev);
+    */
 
    /* If we should forward/feed event using parent class method, then
     * just do it and do NOT create an animator. See _view_smart_mouse_up().
-- 
1.7.4.1

